scripts/main.py

```diff
@@ -1,11 +1,21 @@
+import argparse
```

Comment:
Again it looks like some of these imports are reordered, with blank lines added. Any reason?

Reply:
See #76